install/scripts/aws-up.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+
+set -e
+set -u
+
+# Required env variables:
+: "$AWS_ACCOUNT_ID $AWS_ACCESS_KEY_ID $AWS_SECRET_ACCESS_KEY"
+
+INSTALL_OPERATOR="${INSTALL_OPERATOR:-yes}"
+
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+KUBERENTES_DIR=${SCRIPT_DIR}/../kubernetes
+
+EKSCTL_ENABLE_CREDENTIAL_CACHE=1
+export CLUSTER_NAME=substratus
+export REGION=us-west-2
+export ARTIFACTS_REPO_NAME=${CLUSTER_NAME}
+export ARTIFACTS_BUCKET_NAME=${AWS_ACCOUNT_ID}-${CLUSTER_NAME}-artifacts
+export KARPENTER_VERSION=v0.29.2
+export AWS_PARTITION="aws"
+export KARPENTER_IAM_ROLE_ARN="arn:${AWS_PARTITION}:iam::${AWS_ACCOUNT_ID}:role/${CLUSTER_NAME}-karpenter"
+TEMPOUT=$(mktemp)
+
+aws s3 mb s3://${ARTIFACTS_BUCKET_NAME} \
+  --region ${REGION} >/dev/null || true
+
+aws ecr create-repository \
+  --repository-name ${ARTIFACTS_REPO_NAME} \
+  --region ${REGION} >/dev/null || true
+
+# install karpenter: https://karpenter.sh/docs/getting-started/getting-started-with-karpenter/
+curl -fsSL https://raw.githubusercontent.com/aws/karpenter/"${KARPENTER_VERSION}"/website/content/en/preview/getting-started/getting-started-with-karpenter/cloudformation.yaml >$TEMPOUT &&
+  aws cloudformation deploy \
+    --stack-name "Karpenter-${CLUSTER_NAME}" \
+    --template-file "${TEMPOUT}" \
+    --capabilities CAPABILITY_NAMED_IAM \
+    --parameter-overrides "ClusterName=${CLUSTER_NAME}" \
+    --region ${REGION}
+
+envsubst <${KUBERENTES_DIR}/aws/eks-cluster.yaml.tpl >${KUBERENTES_DIR}/aws/eks-cluster.yaml
+eksctl create cluster -f ${KUBERENTES_DIR}/aws/eks-cluster.yaml ||
+  eksctl upgrade cluster -f ${KUBERENTES_DIR}/aws/eks-cluster.yaml
+
+aws iam create-service-linked-role \
+  --aws-service-name spot.amazonaws.com || true
+
+aws eks update-kubeconfig \
+  --region ${REGION} \
+  --name ${CLUSTER_NAME}
+
+# Logout of helm registry to perform an unauthenticated pull against the public ECR
+helm registry logout public.ecr.aws || true
+helm upgrade \
```

Comment:
Full Karpenter install looks to be supported using eksctl from my reading. Any reason to install using helm?

`It will create all the necessary prerequisites outlined in Karpenter's [Getting Started](https://karpenter.sh/docs/getting-started/) section including installing Karpenter itself using Helm`

https://eksctl.io/usage/eksctl-karpenter/#karpenter-support

Reply:
I see what you meant, we don't need the helm install if `eksctl` is doing that same thing for us 👍 Removing those bits.